reports/distributions/contrib.yaml

```diff
@@ -43,6 +43,7 @@ components:
         - prometheus
         - prometheusremotewrite
         - pulsar
+        - rabbitmq
```

Comment:
This is from following the failed CI check instructions from https://github.com/open-telemetry/opentelemetry-collector-contrib/actions/runs/8335437100/job/22811018791?pr=31800:

```
cd cmd/githubgen && go install .
githubgen distributions
Generated code is out of date, please run "make gendistributions" and commit the changes in this PR.
Error: Process completed with exit code 1.
```

Just double check this is right since I noticed `alertmanagerexporter` isn't in this list even though it's also a recently added exporter with _development_ stability. 

If it's correct then I can add `make gendistributions` to CONTRIBUTING.md's list of commands to run when generating a new componen

Reply:
please do, in a separate PR, thanks!

Reply:
given that the component should not be in contrib initially, you can also just remove that line. You'll need to regenerate the README as well.

Reply:
```suggestion
```